css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-font-size-xs: $font-size-9 !default;
+$persona-font-size-sm: $font-size-8 !default;
+$persona-font-size-md: $font-size-7 !default;
+$persona-font-size-lg: $font-size-6 !default;
+$persona-font-size-xl: $font-size-5 !default;
+
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-avatar-border-radius: $border-radius-rounded;
+
+$persona-name-font-size: $font-size-7 !default;
+$persona-name-font-color: $text !default;
+
+$persona-metadata-font-size: $font-size-9 !default;
+$persona-metadata-font-color: $text-subtle !default;
+
+$persona-gap-size: $layout-1 !default;
+
+.persona {
+	display: flex;
+	position: relative;
+	gap: $persona-gap-size;
+
+	.persona-avatar {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		width: 2.35em;
+		height: 2.35em;
+		border-radius: $persona-avatar-border-radius;
+		background-color: $persona-avatar-background-color;
+		color: $persona-avatar-font-color;
+		font-size: $persona-font-size-md;
```

Comment:
Reminder to once again go through and use em sizing with font-size applied at the component level.

Reply:
Updated